app/src/config.ts

```diff
@@ -162,32 +162,32 @@ This value must match exactly one of the values defined in aspectRatios.`,
 		default :
 		{
 			3840 :
-				[
-					{ scaleResolutionDownBy: 12, maxBitRate: 150000 },
-					{ scaleResolutionDownBy: 6, maxBitRate: 500000 },
-					{ scaleResolutionDownBy: 1, maxBitRate: 10000000 }
-				],
+			[
+				{ scaleResolutionDownBy: 4, maxBitRate: 1500000 },
+				{ scaleResolutionDownBy: 2, maxBitRate: 4000000 },
+				{ scaleResolutionDownBy: 1, maxBitRate: 10000000 }
+			],
 			1920 :
-				[
-					{ scaleResolutionDownBy: 6, maxBitRate: 150000 },
-					{ scaleResolutionDownBy: 3, maxBitRate: 500000 },
-					{ scaleResolutionDownBy: 1, maxBitRate: 3500000 }
-				],
+			[
+				{ scaleResolutionDownBy: 4, maxBitRate: 750000 },
+				{ scaleResolutionDownBy: 2, maxBitRate: 1500000 },
+				{ scaleResolutionDownBy: 1, maxBitRate: 4000000 }
+			],
 			1280 :
-				[
-					{ scaleResolutionDownBy: 4, maxBitRate: 150000 },
-					{ scaleResolutionDownBy: 2, maxBitRate: 500000 },
-					{ scaleResolutionDownBy: 1, maxBitRate: 1200000 }
-				],
+			[
+				{ scaleResolutionDownBy: 4, maxBitRate: 250000 },
+				{ scaleResolutionDownBy: 2, maxBitRate: 900000 },
+				{ scaleResolutionDownBy: 1, maxBitRate: 3000000 }
+			],
 			640 :
-				[
-					{ scaleResolutionDownBy: 2, maxBitRate: 150000 },
-					{ scaleResolutionDownBy: 1, maxBitRate: 500000 }
-				],
+			[
+				{ scaleResolutionDownBy: 2, maxBitRate: 250000 },
+				{ scaleResolutionDownBy: 1, maxBitRate: 900000 }
+			],
 			320 :
-				[
-					{ scaleResolutionDownBy: 1, maxBitRate: 150000 }
-				]
+			[
+				{ scaleResolutionDownBy: 1, maxBitRate: 250000 }
+			]
```

Comment:
Why?

Reply:
I think some files got overwritten when i did the merge with develop.